homeassistant/components/homekit/type_media_players.py

```diff
@@ -261,12 +259,10 @@ def __init__(self, *args):
         self.sources = []
 
         # Add additional characteristics if volume or input selection supported
-        self.chars_tv = []
+        self.chars_tv = [CHAR_REMOTE_KEY]
```

Comment:
Don't love this personally, especially since the comment is not really describing the functionality anymore... 

Thoughts on keeping it `[]` and doing a spread:

```py
serv_tv = self.add_preload_service(SERV_TELEVISION, [CHAR_REMOTE_KEY, *self.chars_tv])
```

Reply:
I think its fine how you have it.